com.unity.netcode.gameobjects/Samples/ClientNetworkTransform/Scripts/ClientNetworkTransform.cs

```diff
@@ -26,6 +26,7 @@ public override void OnNetworkSpawn()
 
         protected override void Update()
         {
+            CanCommitToTransform = IsOwner;
```

Comment:
I guess this is needed in addition to the `OnNetworkSpawn` call because this can change between `ONS` and `Update`?   Because this seems a bit odd I think this is worth a comment

Reply:
Also, I guess one could think "why does ClientNT have this setting in Update and ONS but regular NT only has it in ONS

Reply:
Yeah you're right this would have been worth a comment but I already merged the original PR this is just the backport 😭